cypress/support/pages/contactForm.pageObject.js

```diff
@@ -19,11 +19,11 @@ class ContactFormPageObject extends PageObject {
     return cy.contains('.btn', 'Send message');
   }
 
-  typeEmail(email) {
+  typeEmail() {
     this.emailField.type(email, { force: true });
   }
 
-  typeName(name) {
+  typeName() {
     this.nameField.type(name, { force: true });
```

Comment:
The `typeName` method is missing the `name` parameter in its definition. Ensure that the method accepts a `name` argument and uses it in the `type` function.